pkgs/games/openra-mods/mods.nix

```diff
@@ -0,0 +1,293 @@
+{ fetchFromGitHub }:
+
+let
+  extraPostFetch = ''
+    sed -i 's/curl/curl --insecure/g' $out/thirdparty/{fetch-thirdparty-deps,noget}.sh
+    $out/thirdparty/fetch-thirdparty-deps.sh
```

Comment:
Fetching stuff during build is impure, so it can't be accepted to nixpkgs.

Reply:
That is most unexpected, given that these exact lines appear in the `openra` package in nixpkgs: https://github.com/NixOS/nixpkgs/blob/master/pkgs/games/openra/default.nix#L24.

Reply:
Ah, I forgot this one thing: You can actually access network from fixed output derivation, and it is pure. Though, I think, this is a bit fishy setup that checkouts git revision and downloads some resources in the same derivation under the same hash.